workloads.go

```diff
@@ -198,8 +237,50 @@ func (tsw *TimeSeriesRead) IsPartitionDone() bool {
 	return false
 }
 
-// Dummy workload generator for range scans
+func (tsw *TimeSeriesRead) Restart() {
+}
+
 type RangeScan struct {
+	TotalRangeCount int
+	RangeOffset	    int
+	RangeCount      int
+	NextRange       int
```

Comment:
How about int64? Maybe applicable also to RangeOffset and RangeCount.

Reply:
These are indexes into the amount of sub-ranges we split the token-space into. `rangeCount` is `int` as well and I don't think a `int64` is necessary, why would somebody want to split the token-space into more than 2b sub-ranges. :)

Reply:
:+1: